qemu/tests/cfg/discard_for_raw_block_target.cfg

```diff
@@ -0,0 +1,15 @@
+- discard_for_raw_block_target:
+    only raw
+    virt_test_type = qemu
+    type = discard_for_raw_block_target
+    start_vm = no
+    required_qemu = [3.0.0, )
+    remove_image = yes
+    images = "test"
+    image_name_test = "images/test"
+    image_format_test = raw
+    convert_target = "target"
+    image_name_target = "/dev/sdb"
```

Comment:
why assign `/dev/sdb` to `image_name_target`?

Reply:
Updated. Thanks.